cmd/minikube/cmd/docker-env.go

```diff
@@ -405,7 +405,7 @@ docker-cli install instructions: https://minikube.sigs.k8s.io/docs/tutorials/doc
 			exit.Error(reason.InternalDockerScript, "Error generating set output", err)
 		}
 
-		if sshAdd {
+		if sshHost {
```

Comment:
We may want to think about this as this is a breaking change. Scripts that were set to use `--ssh-add` will no longer execute this. We should leave the flag in and mark it deprecated in the flag description and tell users to use `--ssh-host` but have this be a `sshHost || sshAdd` to not break it.